sys/kern/kern_proc.c

```diff
@@ -2529,20 +2529,26 @@ sysctl_kern_proc_c18n(SYSCTL_HANDLER_ARGS)
 
 	n = proc_readmem_cap(curthread, p, (vm_offset_t)p->p_c18n_info, &info,
 	    sizeof(info));
+	if (n != sizeof(info)) {
+		error = EFAULT;
+		goto out;
+	}
 	/*
```

Comment:
Maybe axe the comment?  It should have a leading blank line, but at this point it just duplicates the code.

Reply:
That's kind of why I was in favor of just axing the comment.  I thought about rewriting the comment to cover all the error cases instead so you'd have one comment above all 3 error cases.  But the existing comment is just duplicating what the code "says" so isn't worth keeping IMO (it doesn't have extra rationale or meaning beyond the  C statements).